### ISSUE TITLE
Parallel branch - Conda package tiberius

### ISSUE THREAD

**Issue by rlibouba** [Gaius-Augustus/learnMSA#12]
Hello, 
I'm creating the conda package for Tiberius. In the documentation, it is explained that you need to clone the learnMSA repository and use the parallel branch to run tiberius. Do you know when the changes from the parallel branch will be merged to the main branch?

**Comment by felbecker** [comment]
Hi,

the relevant changes are already merged into main. We still need to run some test with Tiberius though.

At the moment, we have very tight schedules and not much time for development. I expect that we will **bring Tiberius to pip within the next weeks**, though. It will automatically install learnMSA (also via pip). The parallel branch was just a temporary solution, please don't put work into a conda release that relies on that.

I would recommend to let a few weeks pass, if you want to create the conda release for Tiberius. It will be much easier once Tiberius can be installed via pip.

Best,
Felix

**Comment by rlibouba** [comment]
Hello Felix, 

Thank you for your reply. Good luck to you.
Have a good day!

Best, 
Romane

**Comment by felbecker** [comment]
Hi Romane,

a version of learnMSA that is compatible with Tiberius is now available via pip.

We made updates to [Tiberius ](https://github.com/Gaius-Augustus/Tiberius) and created a docker/singularity container for it.
Unfortunately, we were not able to make Tiberius work with newer TensorFlow versions. Therefore, installation of Tiberius without docker remains the same complicated procedure as before (only that learnMSA can now be installed via pip).

At the moment, we decided against a Tiberius release on pip or conda, as we see little benefit and many potential problems with installing the older TensorFlow version, that Tiberius depends on, manually.

I hope that helps.

Best,
Felix
